Project-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js

```diff
@@ -1,2 +1,15 @@
-import React from 'react';
-import './Header.css';
+import React from "react";
+import './header.css';
+
+const HeaderTitle = () => {
+  return (
+    <div className="headerTitle">
+      {/* <img className="lambdaLogo" alt="Lambda Logo" src="https://tk-assets.lambdaschool.com/1c1b7262-cf23-4a9f-90b6-da0d3c74a5c6_lambdacrest.png" /> */}
+      <h1>Lambda School</h1>
+      <a>@LambdaSchool . 26 Jan</a>
```

Comment:
Interesting use of the anchor tag. 

It's one way to apply your anchor styling to some text. However, it doesn't work well from an accessibility standpoint. Browsers will interpret this as a link without a href or alt attribute. 

Alternatively, you could use a `span` instead. And then you can place the span inside of the h1 and style it separately while keeping everything on one line.